openwisp_controller/api/urls.py

```diff
@@ -0,0 +1,40 @@
+from django.urls import include, path
+
+from . import views as api_views
+
+urlpatterns = [
+    path(
+        'controller/',
+        include(
```

Comment:
is this nested include really needed or is it the best practice to do this? I've never seen it.

Reply:
Since more than one app's API's are present in this module I tried to keep the URL in the main urls.py as `api/v1/` so if I don't use nesting here then each time I write path I will be required to add `controller` to it, so proceeded with this approach.

Reply:
Maybe, you could move these URLs in a separate file and get urls from the function there like done in [openwisp-radius urls](https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/urls.py#L18).

Reply:
Sure, I will do it in this manner.

Reply:
@atb00ker this is done.